man/kernel-command-line.xml

```diff
@@ -90,6 +90,7 @@
         <term><varname>systemd.default_standard_output=</varname></term>
         <term><varname>systemd.default_standard_error=</varname></term>
         <term><varname>systemd.setenv=</varname></term>
+        <term><varname>systemd.machine_id=</varname></term>
```

Comment:
systemd.default_standard_output= and the other switches understood by PID 1 use underscores, hence I'd stick to that for this case too.

It's a bit chaotic though, we should clean that up eventually.